psi4/driver/procrouting/proc.py

```diff
@@ -2487,11 +2487,7 @@ def run_scf_gradient(name, **kwargs):
     if core.get_option('SCF', 'REFERENCE') in ['ROHF', 'CUHF']:
         ref_wfn.semicanonicalize()
 
-    if hasattr(ref_wfn, "_disp_functor"):
```

Comment:
Not really a problem to do, but I'm not sure what a scf gradient wants to do with a post-scf dispersion correction. Also, are there any tests I should check before/after I fix this?

Reply:
This is nothing more than a definition game. `scfgrad` is responsible for what Psi [defines](https://github.com/psi4/psi4/blob/2dc08848d963a2a031288cbdea229451457aa30e/psi4/driver/procrouting/scf_proc/scf_iterator.py#L673-L720) as the SCF total energy rather than what Psi defines as the DFT total energy. If you have a less ambiguous name for `scfgrad`, you are welcome to suggest it.

I don't know if there are any relevant tests. Just make sure that the dispersion contribution to the gradient appears in the new reference output file.